simphony/io/h5_cuds.py

```diff
@@ -112,6 +113,45 @@ def add_particle_container(self, particle_container):
         self._file.flush()
         return pc
 
+    def add_mesh(self, name, mesh=None):
```

Comment:
now that the high-level objects have a `name` attribute, this method can be updated to `add_mesh(self, mesh)` similar to `add_particle_container`